backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ExpressionImpl.java

```diff
@@ -84,13 +86,13 @@ public Function getFunction() {
     /**
      * <!-- begin-user-doc --> <!-- end-user-doc -->
      *
-     * @generated
+     * @generated NOT
      */
     public Function basicGetFunction() {
-        // TODO: implement this method to return the 'Function' reference
-        // -> do not perform proxy resolution
-        // Ensure that you remove @generated or mark it @generated NOT
-        return null;
+        return this.getOwnedTyping().stream()
+                .filter(Function.class::isInstance)
+                .map(Function.class::cast)
+                .findFirst().orElse(null);
```

Comment:
`orElse(null);` should be on a separated line

Reply:
Please fix here and every same occurrences in this PR

Reply:
Done